packages/autometrics-lib/README.md

```diff
@@ -28,4 +31,5 @@ async function createUser(payload: User) {
 }
 
 const user = autometrics(createUser);
```

Comment:
Not part of this pr but i noticed `user` as a name for the decorated function. My issue with it is that it is a good name for a value not a function. I think it would be better to change this. Perhaps rename it to `create` as opposed to `user`.  Another solution could be to rename the original async function: `createNewUser` or `createUserRaw`.